docker/apps/kona_host.dockerfile

```
@@ -0,0 +1,33 @@
+FROM ubuntu:22.04 AS host-build
+SHELL ["/bin/bash", "-c"]
+
+ARG CLIENT_TAG
+
+# Install deps
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  build-essential \
+  git \
+  curl \
+  ca-certificates \
+  libssl-dev \
+  clang \
+  pkg-config
+
+# Install rust
+ENV RUST_VERSION=1.81.0
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y --default-toolchain ${RUST_VERSION} --component rust-src
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# Clone kona at the specified tag
+RUN git clone https://github.com/op-rs/kona
+
+# Build kona-host on the selected tag
+RUN cd kona && \
+  git checkout $CLIENT_TAG && \
+  cargo build --workspace --bin kona-host --release && \
+  mv ./target/release/kona-host /kona-host
+
+FROM ubuntu:22.04 AS export-stage
```

Can probably also use `scatch` here for the `export-stage` as in the asterisc repro dockerfile.

```suggestion
FROM scratch AS export-stage
```